frontend/src/app/main/refs.cljs

```diff
@@ -184,6 +184,9 @@
 (def options-mode-global
   (l/derived :options-mode workspace-global))
 
+(def default-font
```

Comment:
This ref is only used from a single namespace, so I think it should be defined in that namespace.

Reply:
Ok, no problem

Reply:
I've double checked this and this ref is used in multiple namespaces (3 according to vs code: dashboard, v2-editor and texts).